Rdp cache hunting script (#25305)
demisto/content: Packs/RDPCacheHunting/Scripts/PreProcessImage/README.md

### Context Example

```json
{
    "File": {
        "EntryID": "<EntryIDExample>",
        "Info": "image/jpeg",
        "MD5": "<MD5Example>",
        "Name": "grayscale_IMG_3092",
        "SHA1": "<SHA1Example>",
        "SHA256": "<SHA256Example>",
        "SHA512": "<SHA512Example>",
        "SSDeep": "24576:<SSDeepExample>",
        "Size": 1512471,
        "Type": "JPEG image data, JFIF standard 1.01, aspect ratio, density 1x1, segment length 16, baseline, precision 8, 3024x4032, components 1"
    }
}
```

### Example command

```!PreProcessImage action=sharpened file_entry_id=<file_entry_id_example>```

### Context Example

```json
{
    "File": {
        "EntryID": "<EntryIDExample>",
        "Info": "image/jpeg",
        "MD5": "<MD5Example>",
        "Name": "sharpened_IMG_3092",
        "SHA1": "<SHA1Example>",
        "SHA256": "<SHA256Example>",
        "SHA512": "<SHA512Example>",
        "SSDeep": "24576:<SSDeepExample>",
        "Size": 1757075,
        "Type": "JPEG image data, JFIF standard 1.01, aspect ratio, density 1x1, segment length 16, baseline, precision 8, 3024x4032, components 3"
    }
}
```

> ### Example command

Reviewer: 2 examples are enough, you can remove the rest

This script pre-process an image file from context.

> ## Script Data

Reviewer: Generate the readme with example

---

| **Name** | **Description** |
| --- | --- |
| Script Type | python3 |
| Cortex XSOAR Version | 6.0.0 |

## Inputs

---

| **Argument Name** | **Description** |
| --- | --- |

> | action | The action to perform on the image. |

Reviewer: Add here the possible options as well

> This script pre-process an image file by preforming one of the actions:

Reviewer 2: ```suggestion
This script pre-processes an image file by preforming one of the following actions:
```

- original - return the original image.
- sharpen - process that emphasizes the edges in an image to make it appear clearer and more focused.
- grayscale - convert the image to black, white, and gray colors, in which gray has multiple levels (the value of each pixel represents only the intensity information of the light).

> Each action can also resize the image when given width and height.

Reviewer 2: ```suggestion
Each action can also resize the image when given the width and height.
```

## Script Data

---

| **Name** | **Description** |
| --- | --- |
| Script Type | python3 |
| Cortex XSOAR Version | 6.8.0 |

## Inputs

---

| **Argument Name** | **Description** |
| --- | --- |
| action | The action to perform on the image. |
| image_resize_width | The desired width for resizing the image to. |
| image_resize_height | The desired height for resizing the image to. |
| file_entry_id | The entryID of the file to process. |

## Outputs

---

| **Path** | **Description** | **Type** |
| --- | --- | --- |
| PreProcessImage.file_entry_id_new | The entryID of the created file. | String |

> | PreProcessImage.action | The action that were performed,. | String |

Reviewer 2: ```suggestion
| PreProcessImage.action | The action that was performed. | String |
```
